### PR TITLE
Converted SVGs

### PR DESCRIPTION
**Pull request by pandasa123**
- Addressed #23 
- Reduced `index.js` by **37%**
- Converted all SVGs with [SVGR](https://github.com/smooth-code/svgr)
- Removed `react-svg-inline`, `raw-loader`, `react-inlinesvg`, `svg-inline-loader`
- Removed `moduleNameMapper` from `jest` config in `package.json` since it doesn't have to address raw `svg` files anymore
- Added `lint:fix` command to `package.json` to quickly and easily fix lint issues

### CODE REVIEW

**Comment by pandasa123** [comment]
`index.js` is 37% smaller as well from 30.7 KiB to 19.3 KiB

File: package.json [hunk 1]
```diff
@@ -32,11 +32,8 @@
     "eslint-plugin-react": "^7.13.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.8.0",
-    "raw-loader": "^1.0.0",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
-    "react-inlinesvg": "^0.8.4",
-    "svg-inline-loader": "^0.8.0",
```
Comment (ajwann) [inline package.json]: :+1:

File: src/svg-loader-components/audio.js [hunk 2]
```diff
@@ -1,9 +1,50 @@
 import React from 'react';
-import SVGInline from 'react-svg-inline';
-import image from './audio.svg';
 
 const Audio = props => (
-  <SVGInline svg={image} {...props} />
+  <svg width={55} height={80} fill="#FFF" {...props}>
```
Comment (ajwann) [inline src/svg-loader-components/audio.js]: :+1:  this type of conversion is exactly what I was looking for

File: src/svg-loader-components/audio.js [hunk 3]
```diff
@@ -1,9 +1,50 @@
...
+          values="30;45;13;80;56;72;45;76;34;23;67;30"
+          calcMode="linear"
+          repeatCount="indefinite"
+        />
+      </rect>
+    </g>
+  </svg>
 );
 
 export { Audio };
```
Comment (ajwann) [inline src/svg-loader-components/audio.js]: Export is still good

**Pull request merged**